source/js/view.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * Created by Ksenia Annenko on 24.02.2017.
+ */
+
+/********************** View **********************/
+"use strict";
+
+let viewModel = new ViewModel();
+let occupiedCells;
+let playField = document.getElementById('play_field');
+let rowsInput = document.getElementById('rowsInput');
+let columnsInput = document.getElementById('columnsInput');
+let winLengthInput = document.getElementById('winLengthInput');
+let playerInstanceBlock = document.getElementById('player_instance_block');
+let cells = document.getElementsByClassName("cell");
+let availableSymbolList = viewModel.getAvailableSymbolList();
+
+/************ Starting game ************/
+window.onload = function() {
+    let list = viewModel.getPlayerList();
+    rowsInput.value = viewModel.getRows();
+    columnsInput.value = viewModel.getColumns();
+    winLengthInput.value = viewModel.getWinLength();
+
+    for (let i = 0; i < list.length; i++) {
+        getPlayerInstanceTemplate(list[i]);
+    }
+};
+
+function getPlayerInstanceTemplate(player) {
+    let template = document.getElementById('player_instance_template');
+    let playerInstanceList = document.getElementsByClassName('player_instance');
+
+    if ('content' in document.createElement('template')) {
+        playerInstanceBlock.appendChild(template.content.cloneNode(true));
+
+        let playerElem = playerInstanceList[playerInstanceList.length - 1];
+        let optionSymbol = new Option(player.symbol, player.symbol);
+
+        playerElem.setAttribute('id', player.id);
+        playerElem.querySelector('.form-group .player_name').value = player.name;
+        playerElem.querySelector('.form-group .player_symbol').appendChild(optionSymbol);
+        createSymbolList(playerElem.querySelector('.form-group .player_symbol'));
```

Comment:
This function shouldn't be called here. A better place for it is in the end of the `window.onload` code.